src/base/path.cpp

```diff
@@ -288,7 +295,9 @@ Path operator/(const Path &lhs, const Path &rhs)
     if (lhs.isEmpty())
         return rhs;
 
-    return Path(lhs.m_pathStr + QLatin1Char('/') + rhs.m_pathStr);
```

Comment:
The point was to have resulting path cleaned up as well. E.g. `Path("dir1/dir2") / Path("../foo")` should produce the same result as `Path("dir1/dir2/../foo")` (`m_pathStr` is "dir1/foo").